src/pages/Profile/Profile.css

```diff
@@ -1,18 +1,53 @@
 .profile-page {
+    display: flex;
     height: 100vh;
     width: 100vw;
     display: flex;
     align-items: center;
+    justify-content: center;
+    gap: 50px;
 }
 
-/* TODO: Add your styles here
-         for the Profile page.
-         Hint: The lines above ensure that
-         the page takes up the full width
-         and height of the browser window.
-         This will help you when you get started.
-    */
+.name {
+    font-size: 48px;
+}
 
+.person {
+    width: 200px;
+    height: 200px;
+}
 
+.image-background {
+    background-color: rgb(151, 148, 148);
+    border-radius: 50%;
+    width: 300px;
+    height: 300px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.text {
+    display: flex;
+    flex-direction: column;
+}
 
+.stats {
+    display: flex;
+    flex-direction: column;
+}
+
+.stats-numbers {
+    display: flex;
+    width: 500px;
```

Comment:
having fixed widths is fine for this particular task but in the future when we start doing responsive css we'll need to have different widths for different screen sizes or add some sort of %/em scaling to make this work for multiple types of devices